.github/workflows/deploy.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       ATBC_LOGIN_ENDPOINT: "https://auth.at-bc.ca/api/login"
       ATBC_ENDPOINT: "https://sabc.at-bc.ca/api"
       BYPASS_CRA_INCOME_VERIFICATION: true
+      BYPASS_VALIDATIONS_AT_LOCAL: true
```

Comment:
I believe the "AT_LOCAL" lost the meaning since we are using it also on DEV. Can we rename it to something more expressive like:
BYPASS_APPLICATION_SUBMIT_VALIDATIONS

Reply:
Env variable renamed.